src/PackageManagement.VisualStudio/IDE/EnvDTEProjectName.cs

```diff
@@ -1,49 +1,49 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

```

Comment:
yeah that is my bad, please ignore the commit marked as temp, all these are in that commit.